php/src/consumer-2.php

```diff
@@ -28,7 +30,7 @@
         "regex": "^[0-9a-f]{8}(-[0-9a-f]{4}){3}-[0-9a-f]{12}$",
         "value": "fb5a885f-f7e8-4a50-950f-c1a64a94d500"
     }
-}', 36); // size of uuid
+}', 0); // is size necessary?
```

Comment:
Size is not needed with JSON, because the body is a C string. With binary bodies, we need to know the number of bytes in the body.

I have updated the comments on the function to describe this